src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -2723,6 +2723,12 @@
             }
         }
     },
+
+    "XCVRD_LOG": {
+        "Y_CABLE": {
+            "log_verbosity": "notice"
+        }
+    },
```

Comment:
This table should be at the same level as BANNER_MESSAGE.

Reply:
fixed